components/table/TableRow.jsx

```diff
@@ -1,15 +1,19 @@
 import React from 'react'
 import { Table } from '@oacore/design'
 import { classNames } from '@oacore/design/lib/utils'
+import { observer } from 'mobx-react'
 
 import tableClassNames from './index.css'
 
-const TableRow = React.memo(props => {
+const TableRow = observer(props => {
```

Comment:
I am not sure you need `observer`. Does not it work without?

Reply:
Already tried it has to be there. Observer checks only direct children in render.  :) 

> Note that observer only subscribes to observables used during the own render of the component. So if observables are passed to child components, those have to be marked as observer as well. This also holds for any callback based components.

See docs for more
https://mobx.js.org/refguide/observer-component.html

Reply:
So, there are 2 other options to consider: use decorator or wrap on export clause.